lib/index.js

```diff
@@ -128,6 +122,70 @@ auth0GuardianJS.prototype.start = function start(callback) {
 /**
  * @public
  *
+ * This takes a parameter what transaction.serialize returns to
+ * resume the transaction with auth0-mfa-api
```

Comment:
"Takes a parameter returned from transaction.serialize to resume the transaction with auth0-mfa-api"

Correct me if I'm not interpreting it correctly, though